### PR TITLE
Prac 09 feedback

### PR DESCRIPTION
**Pull request by AxyutitsArtyom**
@amritakhatri
@dianabukeikhanova

### CODE REVIEW

File: prac_09/taxi_simulator.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    display_taxis(taxis)
+    while True:
+        choice = input("Choose taxi: ")
+        if choice.isdigit():
+            choice = int(choice)
+            if 0 <= choice < len(taxis):
+                return taxis[choice]
+        print("Invalid taxi choice")
+        print(f"Bill to date: ${total_bill:.2f}")
+
```
Comment (DianaBukeikhanova) [inline prac_09/taxi_simulator.py:56]: good code

File: prac_09/band.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+    def __str__(self):
+        members_info = ", ".join([str(member) for member in self.members])
+        return f"{self.name} ({members_info})"
+
+    def add(self, member):
+        self.members.append(member)
+
+    def play(self):
+        return "\n".join([member.play() for member in self.members])
```
Comment (DianaBukeikhanova) [inline prac_09/band.py:15]: nice code

File: prac_09/silver_service_taxi_test.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
+from prac_09.silver_service_taxi import SilverServiceTaxi
+
+test = SilverServiceTaxi(100, "test", 2)
+
+test.drive(18)
+print(test.get_fare())
+print(test)
```
Comment (DianaBukeikhanova) [inline prac_09/silver_service_taxi_test.py:7]: Great job

**Comment by amritakhatri** [comment]
Its working well

**Pull request merged**